### ISSUE TITLE
[Feb 2025] Accessibility Audit: Org admin user flows

### ISSUE THREAD

**Issue by DanielSass** [CDCgov/prime-simplereport#8513]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the org admin user flows. Do the page scan as well as any relevant guided tests.
Pages to visit:
* `/app/dashboard`
* `/app/settings/*`

Use cases to test:
* view the analytics dashboard with preset and custom date ranges
* add/edit users
   ⚠ the support admin "Manager users" page shares the same "User information" and "Facility access" tabs as the "Manage users" page for org admins
* create/edit a facility
* view facilities
* manage org
* view patient self-registration settings

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

### Additional context
See tickets from previous audits #5773, #6333

**Comment by emyl3** [comment]
Findings:
- `/dashboard` [dev.simplereport.gov-org-admin-Testing-data-dashboard-2025-02-27.json](https://github.com/user-attachments/files/19013860/dev.simplereport.gov-org-admin-Testing-data-dashboard-2025-02-27.json)

**Comment by emyl3** [comment]
No issues created -- ready for review
